AK/Bitmap.h

```diff
@@ -157,6 +157,47 @@ class Bitmap {
         return -1;
     }
 
+    int find_up_to_n_unset(int n, int& region_size) const
```

Comment:
It's not clear from the name that this will succeed even if it doesn't find enough unset bits in a sequence.
Can we clear it up with a more descriptive name?

```cpp
int find_longest_range_of_unset_bits(int max_length, int& found_range_size);
```